src/main/java/org/opensearch/security/user/UserService.java

```diff
@@ -276,4 +279,35 @@ public static void saveAndUpdateConfigs(final String indexName, final Client cli
             throw ExceptionsHelper.convertToOpenSearchException(e);
         }
     }
+
+    public void removeNonServiceAccounts(SecurityDynamicConfiguration<?> configuration) {
```

Comment:
Due to  the refactoring, I changed the method names to better describe their new behaviour